src/pages/index.js

```diff
@@ -67,6 +67,19 @@ const IndexPage = ({data}) => {
 
         <StatesList stateSlugs={stateSlugs}/>
       </div>
+
+      <hr className="mt-7" />
+
+      <section className="text-center mb-8">
+        <br className="d-none d-lg-block" />
+        <div className="h1 font-weight-bold">Ready to do your part now?</div>
+
+        <p className="h4 mt-4">
+          Learn how to <strong>electrify your own machines</strong> and <strong>pass local policy</strong> to electrify the rest
+          <br className="d-none d-lg-block" />
+          <a class='btn btn-lg btn-outline-secondary my-4' href='/take-action'>Take Action</a>
```

Comment:
I wonder if we should use a different button style or tweak this one, as the outline secondary button isn't super prominent (this also applies to the homepage):

![image](https://user-images.githubusercontent.com/3187531/177679131-14a9a84b-9983-4977-8e5a-e44c018fbb41.png)

Reply:
went with the `btn-success` here
<img width="947" alt="Screen Shot 2022-07-07 at 4 53 25 PM" src="https://user-images.githubusercontent.com/919583/177877773-cc5c59a1-470d-4b72-bc07-6d34e99022c3.png">